src/pages/library/Showcase.js

```diff
@@ -16,104 +18,337 @@ import * as Style from 'src/libs/style'
 import * as Utils from 'src/libs/utils'
 
 
+// Description of the structure of the sidebar. Case is preserved when rendering but all matching is case-insensitive.
+const sidebarSections = [
+  {
+    name: 'Getting Started',
+    labels: ['Workflow Tutorials', 'Notebook Tutorials', 'Data Tutorials', 'RStudio Tutorials', 'Galaxy Tutorials'],
+    keepCollapsed: true
+  },
+  {
+    name: 'Analysis Tools',
+    labels: ['WDLs', 'Jupyter Notebooks', 'RStudio', 'Galaxy', 'Hail', 'Bioconductor', 'GATK', 'Cumulus', 'Spark']
+  },
+  {
+    name: 'Experimental Strategy',
+    labels: ['GWAS', 'Exome Analysis', 'Whole Genome Analysis', 'Fusion Transcript Detection', 'RNA Analysis', 'Machine Learning',
+      'Variant Discovery', 'Epigenomics', 'DNA Methylation', 'Copy Number Variation', 'Structural Variation', 'Functional Annotation']
+  },
+  {
+    name: 'Data Generation Technology',
+    labels: ['10x analysis', 'Bisulfate Sequencing']
+  },
+  {
+    name: 'Scientific Domain',
+    labels: ['Cancer', 'Infectious Diseases', 'MPG', 'Single-cell', 'Immunology']
+  },
+  {
+    name: 'Datasets',
+    labels: ['AnVIL', 'CMG', 'CCDG', 'TopMed', 'HCA', 'TARGET', 'ENCODE', 'BioData Catalyst', 'TCGA', '1000 Genomes', 'BRAIN Initiative',
+      'gnomAD', 'NCI', 'COVID-19']
+  },
+  {
+    name: 'Utilities',
+    labels: ['Format Conversion', 'Developer Tools']
+  },
+  {
+    name: 'Projects',
+    labels: ['HCA', 'AnVIL', 'BRAIN Initiative', 'BioData Catalyst', 'NCI']
+  }
+]
+
+const uniqueSidebarTags = _.flow([
+  _.flatMap(s => _.map(_.toLower, s.labels)),
+  _.uniq
+])(sidebarSections)
+
 const styles = {
   column: { marginRight: '1.5rem', flex: '1 1 0px', maxWidth: 415 },
   header: {
-    fontSize: 22, color: colors.dark(), fontWeight: 500,
+    fontSize: 19, color: colors.dark(), fontWeight: 'bold',
     marginBottom: '1rem'
+  },
+  sidebarRow: {
+    display: 'flex', justifyContent: 'space-between', alignItems: 'baseline'
+  },
+  nav: {
+    background: {
+      position: 'absolute', left: 0, right: 0, top: 0, bottom: 0,
+      overflow: 'auto', cursor: 'pointer',
+      zIndex: 2
+    },
+    container: state => ({
+      ...(state === 'entered' ? {} : { opacity: 0, transform: 'translate(-2rem)' }),
+      transition: 'opacity 0.2s ease-out, transform 0.2s ease-out',
+      display: 'flex', flexDirection: 'column'
+    }),
+    navSection: {
+      alignItems: 'center', flex: 'none', padding: '1.2rem 0', fontWeight: 700,
+      borderTop: `1px solid ${colors.dark(0.35)}`
+    }
+  },
+  pill: {
+    width: '4.5rem', padding: '0.25rem', fontWeight: 500, textAlign: 'center',
+    border: '1px solid', borderColor: colors.dark(0.25), borderRadius: '1rem',
```

Comment:
Actually, this doesn't work along with the idea of merging in the highlight style because the highlight only changes the border color. I could repeat the `1px solid` part in the highlight style, but I'd rather keep the conditionally overridden border color separate instead.